Specs/Number/French/PercentModelPercentMode

```diff
@@ -0,0 +1,157 @@
+[
+  {
+    "Input": "100%",
+    "NotSupported": "javascript, python",
```

Comment:
Please mark all new specs as
    "NotSupported": "dotnet, javascript, python, java",
 
Each spec case should have an annotation like this. This is to avoid breaking the build with new cases that are not yet supported.

Reply:
If this build doesn't fail, you can skip adding in this one. But please add them in all future PRs.